211/3/restaurant/app/views/restaurants/show.erb

```diff
@@ -0,0 +1,44 @@
+<div class="text-center" >
+  <div class="media">
+    <div class="media-left">
+      <%= image("../#{@restaurant.photo}", {width: 500}) %>
+    </div>
+    <div class="media-body">
+      <h2 class="media-heading"><%= @restaurant.name %></h2>
+      <p> <%= @restaurant.description.gsub(/\n/, '<br>') %></p>
+      <h3>RATING: <%= @restaurant.rating %></h3>
+      <iframe width="400" height="200" id="gmap_canvas" src="https://maps.google.com/maps?q=<%= @restaurant.latitude %>%2C%20<%= @restaurant.longtitude%>&t=&z=13&ie=UTF8&iwloc=&output=embed" frameborder="0" scrolling="no" marginheight="0" marginwidth="0"></iframe>
+    </div>
+  </div>
+</div>
+
+<div >
+  <h3>Reviews:</h3>
+  <% @restaurant.reviews.each do |review|%>
+    <p><strong> <%= review.user.login %></strong> gives us  <strong><%=  review.mark %></strong> points and wrote:
+    <%= review.description %></p>
+  <% end %>
+</div>
+
+<div class="text-center">
+<% if session[:user_id] %>
+  <form action="/restaurants/<%= @restaurant.id %>/reviews" method="POST">
+    <p>Hi, <%= User.find(session[:user_id]).login %>! Rate us </p>
```

Comment:
1. you should not use queries in templates. Templates are only to render some data. Use `current_user` from controller to ask for login.
2. login is actually a verb. Use `name` to get property of name